pages/common/ftp.md

```diff
@@ -0,0 +1,23 @@
+# ftp
+
+> Internet File Transfer Program
+
+- Connect to a FTP server:
+
+`ftp {{url}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off prompts for each process:
```

Comment:
What does "each process" refer to? Processes haven't been mentioned here at all until now.